database/gov.go

```diff
@@ -16,17 +17,18 @@ import (
 
 // SaveGovParams saves the given x/gov parameters inside the database
 func (db *Db) SaveGovParams(params *types.GovParams) error {
-	depositParamsBz, err := db.EncodingConfig.Marshaler.MarshalJSON(&params.DepositParams)
+
+	depositParamsBz, err := json.Marshal(&params.DepositParams)
 	if err != nil {
 		return err
 	}
 
-	votingParamsBz, err := db.EncodingConfig.Marshaler.MarshalJSON(&params.VotingParams)
+	votingParamsBz, err := json.Marshal(&params.VotingParams)
```

Comment:
Why are you changing the encoder everywhere here?

Reply:
db.EncodingConfig.Marshaler.MarshalJSON do not completable with bdjuno type. It only works when the types is from cosmos-sdk. Using json.Marshal give out exact same json string as cosmos-sdk one.